scapy/main.py

```diff
@@ -58,13 +54,17 @@ def _probe_config_file(cf):
     else:
         return cf_path
 
-def _read_config_file(cf):
+def _read_config_file(cf, _globals=globals(), _locals=locals(), interactive=True):
```

Comment:
Could you add a docstring and explain what the kw args do ?

Reply:
Done